content/docs/tasks/security/rbac-groups/index.md

```diff
@@ -122,6 +122,8 @@ communicating with `httpbin`.
     EOF
     {{< /text >}}
 
+    Policies take effect after some time, please wait at least one minute.
```

Comment:
Is there a way to check when the policies have taken effect? If so, document it, and update all the other places in this doc that request the user to wait for policies to take effect.